lib/utils.ts

```diff
@@ -15,8 +16,13 @@ export const numToDays = [
   'Saturday',
 ]
 
-export const getRandomBrandColor = () =>
-  BRAND_COLORS[Math.floor(Math.random() * BRAND_COLORS.length)]
+export const getRandomBrandColor = () => {
+  const reminders = useReminderStore((state) => state.reminders)
+  const idx = reminders.length
+    ? reminders.length % BRAND_COLORS.length
+    : Math.floor(Math.random() * BRAND_COLORS.length)
+  BRAND_COLORS[idx]
```

Comment:
Let's leave `getRandomBrandColor` as is. Instead, inside the create reminder screen we'll just assign the color to `BRAND_COLORS[reminders.length % BRAND_COLORS.length]`.